dubbo-metadata-report/dubbo-metadata-report-api/src/main/java/org/apache/dubbo/metadata/integration/MetadataReportService.java

```diff
@@ -86,6 +86,24 @@ public static MetadataReportService instance(Supplier<URL> metadataReportUrl) {
         return metadataReportService;
     }
 
+    private static boolean isInterfaceAllowed(Class<?> tClass, ClassLoader classLoader) {
```

Comment:
please remove this method since you do not call it anywhere

Reply:
I have removed